src/components/TodoAppender.jsx

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+function TodoAppender({ onAddTodo }) {
+  const [todoText, setTodoText] = useState('');
+
+  function handleChange({ target }) {
+    setTodoText(target.value);
+  }
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    if (!todoText) {
+      return;
+    }
+
+    onAddTodo(todoText);
+    setTodoText('');
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input type="text" placeholder="할 일을 입력해 주세요" value={todoText} onChange={handleChange} />
```

Comment:
```suggestion
      <input 
        type="text"
        placeholder="할 일을 입력해 주세요" 
        value={todoText} 
        onChange={handleChange} 
      />
```

따로 줄바꿈하시는 기준이 있으신가요?
저는 속성이 2개 이상일 때는 줄바꿈해주면 코드가독성이 더 좋다고 생각하고 코드를 정리해주는 편이에요.
코드 가독성을 고려한 자기만의 코드 정리 기준을 세우시면 좋아요.
비교해보시고 판단해보시면 좋을 것 같네요 ;)

Reply:
오 좋은 리뷰 감사합니다!!
저는 2개는 너무 적은거 같아서 3개 이상일 경우 코드가 더 깔끔하다고 생각해서 3개 이상인 경우로 적용했습니다.